Products/ZenCollector/scheduler.py

```diff
@@ -154,7 +157,14 @@ def late(self):
         """
         Called whenever this task is late and missed its scheduled run time.
         """
-        # TODO: report an event
+        try:
+            # send event only for missed runs on devices.
+            self.task._eventService.sendEvent(
+                STATUS_EVENT, device=self.task._devId,
+                summary="Missed run on {}".format(self.task._manageIp),
+                component="zencommand", severity=Event.Warning)
```

Comment:
this is the place where clear events are sent:
https://github.com/zenoss/zenoss-prodbin/blob/develop/Products/ZenRRD/zencommand.py#L631